miio/protocol.py

```diff
@@ -217,3 +225,225 @@ def _decode(self, obj, context, path):
         Checksum(Bytes(16), Utils.md5, Utils.checksum_field_bytes),
     ),
 )
+
+
+class Protocol:
```

Comment:
Could you move this to its own file (thus, avoid moving this again soonish and polluting the git logs), I'd say `miioprotocol.py` would be a good candidate.

Reply:
✅ Moved `Protocol` to `miioprotocol.py` as `MiIOProtocol`

Reply:
Great, thanks for the effort! :tada: